tox.ini

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[flake8]
+max-line-length = 100
+ignore = W503
+copyright-check = True
+copyright-author = Canonical Ltd.
+select = E,F,W,C
```

Comment:
If you wish to be consistent with other teams, consider putting this in a pyproject.toml with other tool configs and leave the tox file to *just* define environments.

Reply:
Given that we don't have any `pyproject.toml` files in our repositories I prefer having this in our `tox.ini` I also find it a bit more intuitive as this is where flake should be called.